features/docs/gegeven-stap-definities-ruwe-gegevens.feature

```diff
@@ -0,0 +1,175 @@
+# language: nl
+@stap-documentatie
+Functionaliteit: gegevens opgeven met waardentabel
+
+  Regel: heeft {object soort} met de volgende gegevens
+
+    @integratie
+    Abstract Scenario: heeft <relatie> met de volgende gegevens
+      Gegeven de persoon 'P1' met burgerservicenummer '000000012'
+      * heeft <relatie> met de volgende gegevens
```

Comment:
wat is het doel van deze stap definitie? In principe kan een partner worden aangemaakt zonder het zetten van verbintenis_soort en relatie_start_datum

Reply:
Deze stappen helpen om de S&T test zo te maken, als ze in de database staan en zijn gemaakt voor de Gezag API test door Schouwen en Toetsing van RvIG.
De bedoeling is dat we deze test in onze automation kunnen uitvoeren, zodat issues hieruit niet pas bij tests op de RvIG LAP omgeving gevonden worden.
Dit doe ik door export van de gegevens in de database te vullen. Dus alle enigszins relevante gegevens van de persoon, ouders, partners, kinderen en andere categorieën worden exact overgenomen zoals ze in de database staan op de RvIG LAP-omgeving.

Reply:
Maar de oude stap definities werken toch ook? Daar zijn de S&T testen al in geschreven, toch? Wat is dan de meerwaarde om de oude stap definities te herschrijven? Die waren data-driven, dat zijn deze nieuwe stap definities ook

Reply:
Wat we willen is de tests draaien met zowel tags GezagApi en GezagApiDeprecated. Dan kunnen we zien dat in beide varianten de juiste uitkomsten komen. Ik had begrepen dat hiervoor de expressieve stopdefinities nodig waren.

Reply:
Ik was dit vergeten. Ik stel dan voor om dan een aanduiding voor de ouder, kind of partner te gebruiken. Dus bijv. `* heeft een ouder 1 'P2' met de volgende gegevens`. En om te wijzigen kan de stap er dan als volgt uitzien: `En ouder 1 'P2' is gewijzigd naar de volgende gegevens`. Dan kan in de automation met de aanduiding de betreffende relatie worden gevonden

Reply:
Ik heb dit voorstel geïmplementeerd in de laatste commit. Zie: https://github.com/BRP-API/brp-shared/pull/66/commits/65df2b7d855bfba5b219d51e6376469cc73b1d7c.
Ik heb dit met nieuwe create/wijzig/corrigeer relatie functies gedaan, zodat aanroepen naar de bestaande relatie functie nog niet te hoeven worden aangepast

Reply:
@fsamwel: aangepast. Bij de ouder 1 en 2 stappen kan geen aanduiding meer worden opgegeven

Reply:
helemaal blij mee. @MelvLee zullen we deze mergen?

Reply:
@fsamwel, ik heb het approved